setup/pkg/fieldlabs/environment_create.go

```diff
@@ -198,6 +198,7 @@ func (e *EnvironmentManager) mergeWriteTFInstancesJSON(labStatuses []Lab) error
 	}
 
 	for _, labInstance := range labStatuses {
+		u := strings.Split(labInstance.Status.Env.Name, " ")
```

Comment:
I really don't like one-letter variable names -- can we use something more descriptive?

Reply:
Changed from u to firstname

Reply:
Also commented out the force password change.